### ISSUE TITLE
problems about high pixel pictures

### ISSUE THREAD

**Issue by Zhousiyuhit** [a7b23/MazePathFinder#3]
Hi, if I want to train pictures in 200*200*3, do you have some advices for me about how to choose the network weights? Maybe I think 2 conv and 2 fc layers are not satisfied. Thank you very much.

**Comment by Zhousiyuhit** [comment]
sorry, there are some errors about my issue, pictures in 200X200X3.

**Comment by a7b23** [comment]
Hi i think you should increase the number of conv layers and increase the hidden units in fc layers too.

**Comment by Zhousiyuhit** [comment]
Yes, but I know nothing about CNN, can you give me some advices about the parameters in hidden layers? How do you choose the right parameters in your networks? I encountered some problems and don’t know how to continue my works. By the way, I want to use DQN to recognize the pictures  provided from UAV and send them into UGV to guide UGV to find the most optimal path in Mars. Maybe it sounds like a strange idea.
